to-do.html

```diff
@@ -6,11 +6,41 @@
 </head>
 
 <body>
-    <datalist>
 
-            
-    </datalist>
+    <header>
+        <h1>Pjohn's to do list!</h1>
+    </header>
 
+    <button>
+        New List
+    </button>
+
+    <input type="text" id="list-name">
+    <div id="list">
+        <ul>
+            <li> <input type="text" id="l1"></li>
+            <li> <input type="text" id="l2"></li>
+            <li> <input type="text" id="l3"></li>
+            <li> <input type="text" id="l4"></li>
+            <li> <input type="text" id="l5"></li>
+            <li> <input type="text" id="l6"></li>
+            <li> <input type="text" id="l7"></li>
+            <li> <input type="text" id="l8"></li>
+            <li> <input type="text" id="l9"></li>
+            <li> <input type="text" id="l10"></li>
+            <li> <input type="text" id="l11"></li>
+            <li> <input type="text" id="l12"></li>
+            <li> <input type="text" id="l13"></li>
+            <li> <input type="text" id="l14"></li>
+            <li> <input type="text" id="l15"></li>
+            <li> <input type="text" id="l16"></li>
+            <li> <input type="text" id="l17"></li>
+            <li> <input type="text" id="l18"></li>
+            <li> <input type="text" id="l19"></li>
+            <li> <input type="text" id="l20"></li>
+
```

Comment:
remove newline here
* See if you can look into an HTML formatter to help things look clean with consistent indents and removing newlines that are not needed